### ISSUE TITLE
Accessibility: Password strength not announced

### ISSUE THREAD

**Issue by vikrampm1** [woothemes/woocommerce#51837]
**Prerequisites (mark completed items with an [x]):**
- [x] I have checked that my issue type is not listed here https://github.com/woocommerce/woocommerce/issues/new/choose
- [x] My issue is not a security issue, support request, bug report, enhancement or feature request (Please use the link above if it is).

### Issue Description:

Screen readers do not announce the password strength as it is not programatically associated with the password field and it is set to "polite" rather than given an alert role. Blind users will not know if their password has sufficient strength.

<img width="718" alt="password reset strength message" src="https://github.com/user-attachments/assets/ef2f1c3f-1c32-4297-ae2d-23d232042949">

### Component or Page: 

Account details My Account Page

### Name affected element:

New password field strength meter

### Relevant code snippets:

`<span class="password-input"><input type="text" class="woocommerce-Input woocommerce-Input--password input-text" name="password_1" id="password_1" autocomplete="off"><div class="woocommerce-password-strength good" aria-live="polite" style="">Medium</div><span class="show-password-input display-password"></span></span>`

### Recommended fix:

Add an aria-describedby attribute to the password input referencing the message. Remove the aria-live="polite" attribute from the password strength message and instead at role="alert".
